CHANGELOG.md

```diff
@@ -3,7 +3,64 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 1.1.0
+
+This release contains a mix of features, performance improvements, and bugfixes.
+
+### Topdown and Rego
+
+- rego: Fixing broken `BenchmarkCustomFunctionInHotPath` ([#7312](https://github.com/open-policy-agent/opa/pull/7312)) authored by @johanfylling
+- topdown: Fix out of range error in `numbers.range` built-in ([#7269](https://github.com/open-policy-agent/opa/issues/7269)) authored by @anderseknert
+- topdown: Adding configurable token cache to `io.jwt` token verification built-ins ([#7274](https://github.com/open-policy-agent/opa/pull/7274)) authored by @johanfylling
+- topdown: Reduce allocations in hot path ([#7288](https://github.com/open-policy-agent/opa/pull/7288)) authored by @anderseknert
+- topdown+rego+server: Allow opt-in for evaluating non-det builtins in PE ([#7313](https://github.com/open-policy-agent/opa/pull/7313)) authored by @srenatus
```

Comment:
Link https://github.com/open-policy-agent/opa/issues/6496? Initially, like, last year, this came about as a user issue.